tests/NPSP.py

```diff
@@ -704,7 +687,7 @@ def select_value_from_list(self,list_name,value):
         self.selenium.select_from_list_by_label(loc,value) 
         
     def select_value_from_bge_dd(self,list_name,value):
```

Comment:
What does "dd" stand for in this context? Dropdown? Date Dialog? Something else?